environments/prod/common/config/main-local.php

```diff
@@ -10,8 +10,8 @@
             'charset' => 'utf8',
         ],
         'mailer' => [
-            'class' => 'yii\swiftmailer\Mailer',
-            'viewPath' => '@common/mail',
+            'class' => 'yii\symfonymailer\Mailer',
+            'useFileTransport' => true,
```

Comment:
Production should not write to files.

Reply:
You mean useFileTransport should be set to false ?